### DISCUSSION TITLE
Logger in Fw vs Svc

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1406] [Q&A]
Hello

I can find logger framework implementation in both Fw and Svc folder. What is the objective and purpose at high level between both.

Svc:
![image](https://user-images.githubusercontent.com/103112583/163337456-5452fff5-4ceb-4652-bc0e-df248f5cc92d.png)

Fw:
![image](https://user-images.githubusercontent.com/103112583/163337582-c32b7a9e-f19c-4e39-ad08-462b53bc755b.png)

**Comment by timcanham** [comment]
The `ActiveLogger` component queues up F Prime events for transmission to a ground system, while `ActiveTextLogger` will print them to stdout of the target computer. Events are described [here](https://nasa.github.io/fprime/UsersGuide/user/cmd-evt-chn-prm.html#events).
